system-addon/content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -155,7 +174,14 @@ class CollapsibleSection extends React.PureComponent {
   }
 }
 
-CollapsibleSection.defaultProps = {Prefs: {values: {}}};
+CollapsibleSection.defaultProps = {
+  document: global.document || {
+    addEventListener: () => {},
+    removeEventListener: () => {},
+    visibilityState: "prerender"
```

Comment:
@sarracini fixed it with this. Seem reasonable?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Document/visibilityState

Reply:
Yup this seems good!